src/Sylius/Bundle/UiBundle/Twig/DataTestAttributeArrayExtension.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\UiBundle\Twig;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+final class DataTestAttributeArrayExtension extends AbstractExtension
+{
+    /** @var string */
+    private $env;
+
+    public function __construct(string $env)
+    {
+        $this->env = $env;
+    }
+
+    /**
+     * @return TwigFunction[]
+     */
+    public function getFunctions(): array
+    {
+        return [
+            new TwigFunction(
+                'sylius_data_test_attribute_array',
+                function (?string $data = null, ?string $attributeValue = null): array {
```

Comment:
```suggestion
                function (string $name, ?string $value = null): array {
```

`$data` should be always passed.

I'm also wondering about `$name` for that instead of `$data`.